src/Loader/CucumberNDJsonAstLoader.php

```diff
@@ -157,6 +158,15 @@ private static function getTables(array $items): array
             static function ($tableJson) {
                 $table = [];
 
+                if (!isset($tableJson['tableHeader'])) {
```

Comment:
In the end I went for this approach - if this situation happens currently, we would have a PHP warning (or potentially an error, if they are captured by the library user), so while this change is not 100% backward compatible, I think it's a good compromise for an ambiguous situation.